docs/log-data-in-reportportal/ImportDataToReportPortal.mdx

```diff
@@ -56,43 +56,85 @@ The endpoint **POST /v1/{projectName}/launch/import** allows importing a launch
 
-**launchName** (String) – Launch name. If this parameter is not specified, the file name will be used as the launch name.
+**startTime** – if launch start time needed, or will be used first time in imported report
 
-**skippedIsNotIssue** (Boolean) – When set to **"true"**, all skipped issues are processed without applying a defect type; when set to **"false"**, skipped issues are processed and marked with the defect type "To Investigate". If the parameter is not set, the default behavior is equivalent to "false".
+If the array of attributes contains the next "skippedIsNotIssue" attribute marked as system, then all test items with SKIPPED status will be processed without applying a "To Investigate" defect type.
 
-:::note
-These parameters are optional and can be used individually, in any combination, or all together.
-:::
+```yaml
+ "attributes": [
+    {
+      "key": "skippedIsNotIssue",
+      "system": true,
+      "value": "true"
+    }
+  ]
+```
+
+When set to "false", test items with SKIPPED status will be processed and marked with the defect type "To Investigate". If the parameter is not set, the default behavior is equivalent to "false".
 
-**Scenario 1**
 
-If you want to mark skipped items as "To Investigate", use the following request:
+**Scenario 1** (Test items with SKIPPED status have "To Investigate" status):
 
 ```yaml
-curl -X 'POST' 'http://localhost:8585/api/v1/testProject/launch/import'
+curl -L -X POST 'http://localhost:8080/api/v1/superadmin_personal/launch/import' \
+-H 'Content-Type: multipart/form-data' \
+-H 'Authorization: Bearer {token}' \
+-F 'file=@Launch.zip;type=application/x-zip-compressed' \
+-F 'launchImportRq="{
+  \"description\": \"Launch description\",
+  \"mode\": \"DEFAULT\",
+  \"name\": \"Launch\"
+}";type=application/json'
 ```
 
-**Scenario 2**
-
-If you don't want to mark skipped items as "To Investigate", use the following request:
+**Scenario 2** (Test items with SKIPPED status don't have "To Investigate" status)
 
 ```yaml
-curl -X 'POST' 'http://localhost:8585/api/v1/testProject/launch/import?skippedIsNotIssue=true'
+curl -L -X POST 'http://localhost:8080/api/v1/superadmin_personal/launch/import' \
+-H 'Content-Type: multipart/form-data' \
+-H 'Authorization: Bearer {token}' \
+-F 'file=@Launch.zip;type=application/x-zip-compressed' \
+-F 'launchImportRq="{
+  \"attributes\": [
+    {
+      \"key\": \"skippedIsNotIssue\",
+      \"system\": true,
+      \"value\": \"true\"
+    }
+  ],
+  \"description\": \"Launch description\",
+  \"mode\": \"DEFAULT\",
+  \"name\": \"Launch\"
+}";type=application/json'
 ```
 
-<media-view thumbnail={require('./img/ImportViaAPI.png')} src="https://youtu.be/SkUBTNBWiqs" alt="Import via API on our test automation results dashboard" type="video" />
+<media-view thumbnail={require('./img/ImportViaAPI.png')} src="https://youtu.be/gCBDlX6dlEE" alt="Import via API on our test automation results dashboard" type="video" />
```

Comment:
Please, update the video with content-type application/json highlight

Reply:
New video was uploaded.
Thank you.